packages/medusa/src/services/order-edit.ts

```diff
@@ -389,6 +392,67 @@ export default class OrderEditService extends TransactionBaseService {
     })
   }
 
+  async removeLineItem(
+    orderEditId: string,
+    lineItemId: string
+  ): Promise<OrderEdit> {
+    return await this.atomicPhase_(async (manager) => {
+      const orderEdit = await this.retrieve(orderEditId, {
+        select: [
+          "id",
+          "created_at",
+          "requested_at",
+          "confirmed_at",
+          "declined_at",
+          "canceled_at",
+        ],
+        relations: ["items"],
+      })
+
+      const isOrderEditActive = OrderEditService.isOrderEditActive(orderEdit)
+
+      if (!isOrderEditActive) {
+        throw new MedusaError(
+          MedusaError.Types.NOT_ALLOWED,
+          `Can not update an item on the order edit ${orderEditId} with the status ${orderEdit.status}`
+        )
+      }
+
+      const deleteLineItem = orderEdit.items.find(
+        (it) => it.original_item_id === lineItemId
+      )
+
+      if (typeof deleteLineItem === "undefined") {
+        return orderEdit
+      }
+
+      await this.lineItemService_
+        .withTransaction(manager)
+        .deleteWithTaxLines(deleteLineItem.id)
+
+      await this.refreshAdjustments(orderEditId)
+
+      const editChange: CreateOrderEditItemChangeInput = {
+        original_line_item_id: lineItemId,
+        type: OrderEditItemChangeType.ITEM_REMOVE,
+        order_edit_id: orderEdit.id,
+      }
+
+      // save change
+      await this.orderEditItemChangeService_
+        .withTransaction(manager)
+        .create(editChange)
+
+      await this.eventBusService_
```

Comment:
question: We should think about that since the create method already emit a CREATED event. should be sufficient not?